pkg/agent/nodeportlocal/portcache/port_table_windows_test.go

```diff
@@ -0,0 +1,108 @@
+//go:build windows
+// +build windows
+
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package portcache
+
+import (
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	"k8s.io/client-go/tools/cache"
+
+	portcachetesting "antrea.io/antrea/pkg/agent/nodeportlocal/portcache/testing"
+	"antrea.io/antrea/pkg/agent/nodeportlocal/rules"
+	rulestesting "antrea.io/antrea/pkg/agent/nodeportlocal/rules/testing"
+)
+
+const (
+	startPort = 61000
+	endPort   = 65000
+)
+
+func newPortTable(mockIPTables rules.PodPortRules, mockPortOpener LocalPortOpener) *PortTable {
+	return &PortTable{
+		PortTableCache: cache.NewIndexer(GetPortTableKey, cache.Indexers{
+			NodePortIndex:    NodePortIndexFunc,
+			PodEndpointIndex: PodEndpointIndexFunc,
+			PodIPIndex:       PodIPIndexFunc,
+		}),
+		StartPort:       startPort,
+		EndPort:         endPort,
+		PortSearchStart: startPort,
+		PodPortRules:    mockIPTables,
+		LocalPortOpener: mockPortOpener,
+	}
+}
+
+func TestRestoreRules(t *testing.T) {
```

Comment:
Can you check if possible to add more tests for other functions? The file's coverage is only 30%+.

Reply:
okay